src/Routes.js

```diff
@@ -14,6 +15,7 @@ class Routes extends React.Component {
         <Switch>
           <Route exact path="/login" component={Login} />
           <Route exact path="/signup" component={Signup} />
+          <Route exact path="/main" component={Main} />
```

Comment:
```suggestion
          <Route exact path="/" component={Main} />
```
main 페이지라면 그냥 / 경로로 써주는 것이 자연스러워보입니다

Reply:
깃 관련해서 초기세팅 변화사항이 있어서 저 멘토님 뵙고 라우터 및 폴더 구조 정리하겠습니다